Http/RefreshAuthenticationFailureResponse.php

```diff
@@ -14,7 +14,41 @@
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
 
-class RefreshAuthenticationFailureResponse extends JsonResponse
+if (80000 <= \PHP_VERSION_ID && (new \ReflectionMethod(JsonResponse::class, 'setData'))->hasReturnType()) {
```

Comment:
B/C layer because in SF6 there are PHP 8 types used on the class, so the eval hack is needed for PHP 7.4 compat.